src/components/ListOrgs.vue

```diff
@@ -272,6 +275,60 @@ const exportAll = async () => {
   exportCsv(exportData, `roar-${activeOrgType.value}.csv`);
 };
 
+const exportOrgUsers = async (orgType) => {
+  try {
+    // Fetch the first page of users to estimate total count
+    const users = await fetchUsersByOrg(activeOrgType.value, orgType.id, ref(10001), ref(0), orderBy);
+
+    if (!users || users.length === 0) {
+      toast.add({
+        severity: 'error',
+        summary: 'Export Failed',
+        detail: 'No users found for the organization.',
+        life: 3000,
+      });
+      return;
+    }
+
+    if (users.length >= 10000) {
+      toast.add({
+        severity: 'error',
+        summary: 'Export Failed',
+        detail: 'Too many users to export. Please filter the users by selecting a smaller org type.',
+        life: 3000,
+      });
+      return;
+    }
+
+    const computedExportData = users.map((user) => ({
+      Username: _get(user, 'username'),
+      Email: _get(user, 'email'),
+      FirstName: _get(user, 'name.first'),
+      LastName: _get(user, 'name.last'),
+      Grade: _get(user, 'studentData.grade'),
+      Gender: _get(user, 'studentData.gender'),
+      DateOfBirth: _get(user, 'studentData.dob'),
+      UserType: _get(user, 'userType'),
+    }));
```

Comment:
Can we export all of the available data that we might have? Some of it will be null. But we should include it if we can. E.g., `ell_status`, `frl_status`, etc.

Reply:
done!